l1-contracts/src/core/libraries/crypto/SignatureLib.sol

```diff
@@ -4,13 +4,21 @@ pragma solidity ^0.8.27;
 
 import {Errors} from "@aztec/core/libraries/Errors.sol";
 
+// Attestation Signature
 struct Signature {
-  bool isEmpty;
   uint8 v;
   bytes32 r;
   bytes32 s;
 }
 
+// A committee attestation can be made up of a signature and an address.
```

Comment:
Should it not be a signature *or* an address to make it more clear. This sounds like there might be both.